### PR TITLE
Deal with NaN for Enum variables

### PR DESCRIPTION
**Pull request by benjello**
Thanks for contributing to OpenFisca-Survey-Manager! Remove this line, as well as any other in the following that don't fit your contribution  :)

#### Technical changes

- Deal with Nan in Enum variables

### CODE REVIEW

File: openfisca_survey_manager/scenarios.py [hunk 1]
```diff
@@ -1597,6 +1597,12 @@ def init_variable_in_entity(simulation, entity, variable_name, series, period):
         )
 
     if enum_variable_imputed_as_enum:
+        if series.isnull().any():
+            log.debug('There are {} NaN values for {} non NaN values in variable {}'.format(
```
Comment (nikhilwoodruff) [inline openfisca_survey_manager/scenarios.py:1601]:
```suggestion
            log.debug('There are {} NaN values ({}% of the array) in variable {}'.format(
```

Minor and technically equivalent, but might be slightly nicer.

**Review by nikhilwoodruff — APPROVED** [review]
Looks nice, @benjello.

File: openfisca_survey_manager/scenarios.py (hunk 1 above)
Comment (benjello) [reply openfisca_survey_manager/scenarios.py:1601]: We need also to change the next lines right @nikhilwoodruff ?

File: openfisca_survey_manager/scenarios.py [hunk 2]
```diff
@@ -1597,6 +1597,12 @@ def init_variable_in_entity(simulation, entity, variable_name, series, period):
...
+                series.isnull().sum(), series.notnull().sum(), variable_name))
```
Comment (nikhilwoodruff) [inline openfisca_survey_manager/scenarios.py:1602]:
```suggestion
                series.isnull().sum(), series.isnull().mean() * 100, variable_name))
```

File: openfisca_survey_manager/scenarios.py (hunk 1 above)
Comment (nikhilwoodruff) [reply openfisca_survey_manager/scenarios.py:1601]: Yes, commented.

**Pull request merged**